gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql

```diff
@@ -0,0 +1,375 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql */
+
+DO $$
+BEGIN
+	IF (SELECT a.attrelid IS NULL
+		FROM pg_class c
+		LEFT JOIN pg_namespace n ON n.oid = c.relnamespace
```

Comment:
Why is there a LEFT JOIN here? Can there be a table without a schema? Moreover, the condition explicitly indicates the scheme and the LEFT JOIN loses its meaning.